roles/bridge-setup/tasks/main.yml

```diff
@@ -1,9 +1,31 @@
 ---
 
-- name: Install bridge-utils
+- name: Install bridge-utils and iptables
   yum:
-    name: bridge-utils
+    name: "{{ item }}"
     state: latest
+  with_items:
+    - bridge-utils
+    - iptables
+    - iptables-services
+
+- name: enable iptables
```

Comment:
Hrmmm.... since we're using CentOS 7.4, you should probably switch this to using `firewalld` and then use the firewalld module in Ansible to manage the port creation:

http://docs.ansible.com/ansible/latest/firewalld_module.html

Reply:
But I'm wondering that kube-proxy work with `firewalld`. I use iptables because kube-proxy uses iptables, so I suppose we should use iptable command still.
Or should I wait to commit the patch until kubernetes support  firewalld? :laughing:

Reply:
Isn't firewalld just a wrapper for iptables? Like, iptables is still the underlying system for the firewall, it's just managed with a dedicated front end now right?

Reply:
I understand that firewalld just sends iptables, but it does not said that firewalld is just a command replacement. 
Additionally, there is much information with kube-proxy/iptables than with kube-proxy/iptables, so many user know about only iptables still and may not know that 1:1 command mapping between iptables/firewalld-cmd (at least, I don't know).

If you suppose we must support CentOS 7.4 feature, please close this PR and file another issue with bridge/firewalld (oh, I missed SELinux, so please add it!)

Reply:
Fix this as you see fit to move forward, and go ahead and open any other issues that were noted here so we can address them later.

Reply:
I've change firewalld-cmd but I cannot use firewalld module due to unsupported option in ansible.